lighthouse-core/scripts/update-report-fixture.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+##
+# @license Copyright 2018 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+##
+
+node lighthouse-cli/test/fixtures/static-server.js &
+
+node lighthouse-cli/index.js --output=json http://localhost:10200/dobetterweb/dbw_tester.html > ./lighthouse-core/test/results/sample_v2.json
```

Comment:
i'd recommend using --output_path instead. it will help avoid any logging ending up there.

also with > the file is cleared the second this command starts, whereas with output_path it's just changed when everything is done.

Reply:
sure will do